ui_framework/components/info_panel/_info_panel.scss

```diff
@@ -1,8 +1,59 @@
 .kuiInfoPanel {
-  padding: 20px;
+  padding: 14px 20px 18px;
   line-height: $lineHeight;
+  border: 2px solid;
+}
+
+/**
+ * 1. TODO: Pick a hex value instead of making these colors translucent.
+ */
+.kuiInfoPanel--info {
+  border-color: rgba($infoColor, 0.25); /* 1 */
+}
+
+.kuiInfoPanel--success {
+  border-color: rgba($successColor, 0.25); /* 1 */
 }
 
 .kuiInfoPanel--warning {
-  background-color: #FFF6E4;
+  border-color: rgba($warningColor, 0.25); /* 1 */
+}
+
+.kuiInfoPanel--error {
+  border-color: rgba($errorColor, 0.25); /* 1 */
+}
+
+/**
+ * 1. Align with first line of title text if it wraps.
```

Comment:
Consider numbering this comment something other than `1` so it doesn't clash with the `1` above.